i18n/en.i18n.json

```diff
@@ -84,6 +84,7 @@
     // housing
   "Acknowledge save effects": "I acknowledge that all saved modifications are immediately visible on the community's page.",
   "Create your own community": "Couldn't find your community? Why not create it yourself?",
+  "join request text": "If you own a parcel in this building, you can apply to join by pressing the button below.",
```

Comment:
a text szonak ebben a fajlban nincs semmi informacio tartalma, ezert nem javaslom. (2. fejezet szerint) -- ebben a fajlban minden egy text
Es az a jelenlegi konvencio hogy ha angolra is leforditod akkor ugy irjuk hogy egybe camel case-ben, jelolve hogy ez csak egy szimbolum, es angolra is forditani kell
Ennek tukreben a nev javalaslat joinRequestInstructions lenne

Reply:
Jogos. Furán is nézett ki... :)